src/charm.py

```diff
@@ -53,17 +166,17 @@ def _add_mongodb_org_repository(self):
                 return
 
     def _install_apt_packages(self, packages: list):
```

Comment:
```suggestion
    def _install_apt_packages(self, packages: List[str]):
```

where list is imported as `from typing import List`